docs/docs/responses.mdx

```diff
@@ -10,7 +10,9 @@ abstract: Responses are messages that your assistant sends to the user. A respon
 Responses go under the `responses` key in your [domain](./domain.mdx) file. Start each
 response name with `utter_`.
 
-Here is an example of a domain file with two responses called `utter_greet` and `utter_bye`:
+Responses go under the `responses` key in your domain file or in a separate "responses.yml" file. Each response name should start with `utter_`.
+This response name is the key you should list your response under.
```

Comment:
not sure this was intentional, you're duplicating content from above and there's no reference to the example below